### DISCUSSION TITLE
Suggestion - Maintaining a single installation path

### DISCUSSION THREAD

**Discussion by olihough86** [oobabooga/text-generation-webui#216] [Ideas]
Feel free to tell me I'm wrong but, I've seen quite a few issues get raised and there's always confusion on how did you install it etc.

Just a suggestion but maybe it would make life easier going forward to choose an install path and stick to it (Conda ftw) and phase out 1 click installers etc. I mean the conda install is literally copy paste.

Any kind of virtual python env is fine, though Conda is the most friendly imho, it's not gonna screw with peoples base python packages, and they are essentially disposable, screw up, delete, re create.

Conda is also cross platform so no issues there and should huge breaking changes arrive it's easy to say "ok ditch your env and start again" 

KISS ( Keep It Simple Stupid) I find is often the best path forward.

**Comment by gruevy** [comment]
As a complete newb I found the one click installer more convenient than conda

**Comment by vdruts** [comment]
I have an use conda venv's and for what ever reason this install kept giving me some SSL error which I simply could not fix or figure out.

**Reply by Duemellon** [reply]
same
